src/core/remote.js

```diff
@@ -2277,7 +2281,7 @@ Remote.prototype.requestRipplePathFind = function(options_, callback_) {
       destination_account: options_.dst_account,
       destination_amount: options_.dst_amount,
       source_currencies: options_.src_currencies
-      }, options_);
+    }, options_);
   } else {
     _.merge(options, makeOptions(
       'ripple_path_find',
```

Comment:
Is that a problem? It's unclear whether it's supposed to refer to the rippled name, which is `ripple_path_find`.